src/inject.js

```diff
@@ -58,9 +58,7 @@
   function installChatterino() {
     if (matchChannelName(window.location.href)) {
       showingChat = true;
-      if (settings.replaceTwitchChat) {
-        replaceChat();
-      }
+      replaceChat();
```

Comment:
The chat shouldn't be replaced if the setting is off (that's the whole point of the setting). With this change it will always be replaced, so users can't vote on polls for example.